src/main/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParser.java

```diff
@@ -548,7 +548,8 @@ private boolean parseOption(String key, final String stringValue, final boolean
             } else {
                 value = constructFromString(getUnderlyingType(optionDefinition.field), stringValue);
             }
-
+            // TODO - Picard legacy parsing is not taking into account range within options
+            // TODO - should we add support here?
```

Comment:
I don't think we want to invest in adding support for these to the legacy parser. Perhaps just add some code to log a warning saying its unsupported and not enforced if a value is specified. (We don't want to throw since the new parser will handle them, and we want to be able to switch back and forth between the two parsers).